### ISSUE TITLE
hello pls help me

### ISSUE THREAD

**Issue by Averox402** [dalaranwow/dalaran-wow#5962]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. I am in the middle of the quest : The Touch of zanzil and i cant find leaded Vial 2 of the rouge sellers are Poison suppliers and they only selling poison and nothing else. (i am new at wow) i am at lvl 29 and my name is Pinhasdog and my acc name is averox
![World of Warcraft 15_06_2021 21_03_15](https://user-images.githubusercontent.com/85841921/122101754-5c216d00-ce1d-11eb-8515-a7c6000200b7.png)
![World of Warcraft 15_06_2021 21_03_05](https://user-images.githubusercontent.com/85841921/122102085-b91d2300-ce1d-11eb-94e5-f686ab7bc521.png)

2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![Uploading World of Warcraft 15_06_2021 21_03_05.png…]()
